CI/physmon/physmon.py

```diff
@@ -200,10 +202,10 @@ def run_ckf_tracking(truthSmearedSeeded, truthEstimatedSeeded, label):
             s,
             trackingGeometry,
             field,
-            CKFPerformanceConfig(ptMin=400.0 * u.MeV, nMeasurementsMin=6),
-            TrackSelectorRanges(
+            TrackSelectorConfig(
                 loc0=(-4.0 * u.mm, 4.0 * u.mm),
                 pt=(500 * u.MeV, None),
+                nMeasurementsMin=6,
```

Comment:
I was wondering if this is still bit too conservative. 
nMeas = 2 (3 with time) for a pixel hit. nMeasMin == 6 would mean just >=3 hits if pixel hits are associated (>=2 if time is present). Perhaps increase this to 8-10 to allow for longer tracks?